src/frontend/src/utils/iiConnection.ts

```diff
@@ -43,7 +43,6 @@ import {
 } from "@dfinity/identity";
 import { Principal } from "@dfinity/principal";
 import { isNullish, nonNullish } from "@dfinity/utils";
-import * as tweetnacl from "tweetnacl";
```

Comment:
Wasn't this in the dependencies of package.json?

Reply:
It was transitively available and never accurately reflected in the `package.json`.